### DISCUSSION TITLE
生产环境3个be，表是3副本，挂掉一个be后 就不能读写了

### DISCUSSION THREAD

**Discussion by echobigdata** [apache/doris#35793] [A - General / Q&A]
生产环境doris版本2.0.2 
3be，3副本，停一台be就报错，试了好几次都无法查询。副本看起来也都是健康的。没上生产前测试高可用也没发现问题，不知道是哪里配置不对了嘛
 detailMessage = 30110458 have no queryable replicas. err: 30110459's backend 10025 does not exist or not alive
 
![image](https://github.com/apache/doris/assets/97231854/32604ab3-da3a-469c-99a1-4a8032dd6ef1)

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
